android/app/build.gradle

```diff
@@ -53,10 +53,8 @@ android {
   compileSdkVersion 31
 
   defaultConfig {
-    // 19+ required by the webview_flutter plugin
-    minSdkVersion 19
-    targetSdkVersion 30
-    multiDexEnabled true
+    minSdkVersion 21
```

Comment:
Is there a reason to bump the min sdk?
Less than 1% of users (~20) are on android 4.4 (api 19), so it's not a big deal but if we don't have any benefit from it we should probably leave it as is.

Reply:
Less compatible issues, less dependencies depends on.

Reply:
That's true, I will test it when I have the time to make sure nothing broke

Reply:
We use `minSdk = 23` & `targetSdk = 31` for days, it should be same as sdk 30 in perssmions and files.